crates/oxc_linter/src/rules/no_caller.rs

```diff
@@ -0,0 +1,77 @@
+use oxc_ast::{ast::MemberExpression, AstKind, Span};
+use oxc_diagnostics::{
+    miette::{self, Diagnostic},
+    thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint(no-caller): Disallow the use of arguments.caller or arguments.callee")]
+#[diagnostic(severity(warning))]
```

Comment:
This is a type error, can you add a help message?

Chrome reports

> Uncaught TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them

I think copy by verbatim should be ok here.

Reply:
Is [005d507](https://github.com/Boshen/oxc/pull/178/commits/005d5075e13888042cb71bd406d402843dc92ae6) ok?